fe/src/main/java/org/apache/doris/external/EsIndexState.java

```diff
@@ -76,66 +68,45 @@ public TNetworkAddress randomAddress(Map<String, EsNodeInfo> nodesInfo) {
         EsNodeInfo[] nodeInfos = (EsNodeInfo[]) nodesInfo.values().toArray();
         return nodeInfos[seed].getPublishAddress();
     }
-    
-    public static EsIndexState parseIndexStateV55(String indexName, JSONObject indicesRoutingMap, 
-            JSONObject nodesMap, 
-            JSONObject indicesMetaMap, PartitionInfo partitionInfo) throws AnalysisException {
+
+    public static EsIndexState parseIndexState(String indexName, JSONObject nodesMap,
```

Comment:
EsIndexState maybe need a better name

Reply:
Renaming EsIndexState causes multiple referenced classes to be modified，Is it really necessary to rename it？